main.go

```diff
@@ -13,7 +13,16 @@ func main() {
 	// contention on the main application
 	runtime.GOMAXPROCS(1)
 
-	app, configErr := core.LoadApp()
+	subcommand, params := core.GetArgs()
+	if subcommand != nil {
+		err := subcommand(params)
```

Comment:
I feel like there's a little misdirection/in-and-out going on here but, hey, it's way cleaner because of it! I like the use of returning a func and combining the args.